contents/handbook/engineering/support-hero.md

```diff
@@ -200,9 +211,14 @@ Every team has a Secondary on-call rotation. Unlike support hero, you are still
 
 ### Schedules
 
-- [Secondary - App East](https://posthog.pagerduty.com/schedules#PXUZ9XL)
-- [Secondary - App West](https://posthog.pagerduty.com/schedules#P04FUTJ)
-- [Secondary – Platform Ingestion](https://posthog.pagerduty.com/schedules#PM8YSH8)
-- [Secondary - Platform Infrastructure](https://posthog.pagerduty.com/schedules#P78OOWZ)
+- [Secondary - Product analytics](https://posthog.pagerduty.com/schedules#PXUZ9XL)
```

Comment:
@EDsCODE updated these. The issue is it's super hard/lots of manual work to keep them in sync with the main support hero. Perhaps we just leave this up to the teams to figure out?